metabase/metabase
# Add embedding options for app nav

Epic https://github.com/metabase/metabase/issues/22529

How to test:
- Set `IFRAMED` to `true` https://github.com/metabase/metabase/blob/43997d63575e72743f156b3a61ca11aabf4e856e/frontend/src/metabase/lib/dom.js#L13
- Open `http://localhost:3000/?top_nav=true`. Make sure the top nav is now visible.
- Try `search=true` to show search in the app bar
- Try `new_button=true` to show the new button in the app bar
- Try `side_nav=false` to hide the side navigation in all cases
- Try `side_nav=true` to show it in all cases when it's shown in regular app

Please note that the parameters are only read after the initial loading of the page.
